react/components/ChildrenContainer.js

```diff
@@ -0,0 +1,127 @@
+} from '../utils/categoryMenuPosition'
+
+import categoryMenu from '../categoryMenu.css'
+
+/**
+ * Component responsible for rendering an array of categories and its respective subcategories
+ */
+export default class ChildrenContainer extends Component {
+  static propTypes = {
+    /** Category to be displayed */
+    items: PropTypes.arrayOf(itemPropType).isRequired,
+    /** Department slug */
+    parentSlug: PropTypes.string,
+    /** Close menu callback */
+    onCloseMenu: PropTypes.func.isRequired,
+    /** Indicates if the children must be shown */
+    isShowing: PropTypes.bool,
+    /** Ref to the parent container. Used to position the children below it */
+    containerRef: PropTypes.object,
+    /** Whether to show second level links or not */
+    showSecondLevel: PropTypes.bool,
+    /** Defines the position of the category menu */
+    menuPosition: PropTypes.oneOf(getMenuPositionValues()),
+    /** Custom styles to item container */
+    containerStyle: PropTypes.object,
+  }
+
+  shouldRenderSecondLevel({ children }) {
+    const { showSecondLevel } = this.props
+    return children && !!children.length && showSecondLevel
+  }
+
+  render() {
+    const {
+      items,
+      menuPosition,
+      containerRef,
+      isShowing,
+      onCloseMenu,
+    } = this.props
+
+    const containerStyle = {
+      top: containerRef && containerRef.offsetTop + containerRef.clientHeight,
+      display: isShowing ? 'flex' : 'none',
+    }
+
+    const containerClasses = classNames('w-100 flex flex-wrap pa0 list mw9', {
+      'justify-start': menuPosition === categoryMenuPosition.DISPLAY_LEFT.value,
+      'justify-end': menuPosition === categoryMenuPosition.DISPLAY_RIGHT.value,
+      'justify-center':
+        menuPosition === categoryMenuPosition.DISPLAY_CENTER.value,
+    })
+
+    const columnItemClasses = classNames({
+      'pl0 pr7': menuPosition === categoryMenuPosition.DISPLAY_LEFT.value,
+      'pr0 pl7': menuPosition === categoryMenuPosition.DISPLAY_RIGHT.value,
+    })
+
+    const firstLevelLinkClasses = classNames(
+      `${
+        categoryMenu.firstLevelLink
+      } db pv4 link no-underline outline-0 tl t-small truncate c-on-base underline-hover`,
+      {
+        pr4: menuPosition === categoryMenuPosition.DISPLAY_LEFT.value,
+        pl4: menuPosition === categoryMenuPosition.DISPLAY_RIGHT.value,
+        ph4: menuPosition === categoryMenuPosition.DISPLAY_CENTER.value,
+      }
+    )
+
+    const secondLevelLinkClasses = classNames(
+      `${
+        categoryMenu.secondLevelLink
+      } db pv3 no-underline outline-0 tl link t-small truncate c-muted-1 underline-hover`,
+      {
+        pr5: menuPosition === categoryMenuPosition.DISPLAY_LEFT.value,
+        pl5: menuPosition === categoryMenuPosition.DISPLAY_RIGHT.value,
+        ph5: menuPosition === categoryMenuPosition.DISPLAY_CENTER.value,
+      }
+    )
+
+    return (
+      items &&
+      !!items.length && (
+        <div
+          className={`${
```

Comment:
You could wrap these classes with the previous suggestion using classnames. I think it would be cooler.